portal/views/portal.py

```diff
@@ -1036,8 +1039,20 @@ def get_reporting_counts():
             counts['interventions'][interv.description] += 1
             if (any(doc.intervention == interv for doc in user.documents)):
                 counts['intervention_reports'][interv.description] += 1
-        for org in user.organizations:
-            counts['organizations'][org.name] += 1
+        if not user.organizations:
+            counts['organizations']['Unspecified'] += 1
```

Comment:
Yep, that 'none of the above' org is already taken into account when looking at the `User.organizations`. This is to handle cases where the user has not yet been assigned to _any_ orgs, _including_ the 'none of the above' org; see Justin's comments in [the PT story here](https://www.pivotaltracker.com/story/show/150063689).